Automattic/pocket-casts-android
# Add UI for audio clips

## Description

This adds UI for audio clip sharing and connects it with the preexisting sharing code. I used a slightly different text for audio sharing description. It is `.mp3` instead of `.wav` because we share MP3s.

## Testing Instructions

1. Start clip sharing.
2. Scroll to the last position. It should display an audio icon.
3. Notice animations for button and top text as yous switch back and forth between last two cards.
4. Share an audio clip.
5. You should see a progress on the loading button.

## Screenshots or Screencast 

A slowed down flow:

![demo](https://github.com/user-attachments/assets/8860a0d0-5be4-4dcc-ac61-9e3ffa945bbe)

## Checklist
 
#### I have tested any UI changes...
